easybuild/easyconfigs/w/worker/worker-1.6.7-intel-2017a.eb

```diff
@@ -0,0 +1,110 @@
+easyblock = 'ConfigureMake'
+
+name = 'worker'
+version = '1.6.7'
+
+homepage = 'https://github.com/gjbex/worker'
+description = """The Worker framework has been developed to help deal with parameter exploration experiments
+ that would otherwise result in many jobs, forcing the user resort to scripting to retain her sanity;
+ see also https://vscentrum.be/neutral/documentation/cluster-doc/running-jobs/worker-framework."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/gjbex/worker/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['350ffa36c792c5733ae202e76cc266648fca1311a835e9d73c98ed4f13a6835f']
+
+tcname = 'intel'
+tcver = '2017a'
+builddependencies = [(tcname, tcver)]
+versionsuffix = '-%s-%s' % (tcname, tcver)
+
+exts_defaultclass = 'PerlModule'
+
+exts_list = [
+    ('Config::General', '2.63', {
+        'source_tmpl': 'Config-General-2.63.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TL/TLINDEN'],
+        'checksums': [
+            '0a9bf977b8aabe76343e88095d2296c8a422410fd2a05a1901f2b20e2e1f6fad',  # Config-General-2.63.tar.gz
+        ],
+    }),
+    ('IO::Stringy', '2.111', {
+        'source_tmpl': 'IO-stringy-2.111.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DS/DSKOLL'],
+        'checksums': [
+            '8c67fd6608c3c4e74f7324f1404a856c331dbf48d9deda6aaa8296ea41bf199d',  # IO-stringy-2.111.tar.gz
+        ],
+    }),
+    ('Text::CSV', '1.33', {
+        'source_tmpl': 'Text-CSV-1.33.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA'],
+        'checksums': [
+            '9c5b8fb9baffd58f02ed2b3f0b6d9a6454198f18a80e7f3a049680ddbdb24115',  # Text-CSV-1.33.tar.gz
+        ],
+    }),
+    ('DBI', '1.637', {
+        'source_tmpl': 'DBI-1.637.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TI/TIMB'],
+        'checksums': [
+            '2557712593e80142c3b50877e00369b6ce78fa26d44edc42156d81a5cdd26bc6',  # DBI-1.637.tar.gz
+        ],
+    }),
+    ('DBD::SQLite', '1.55_03', {
+        'source_tmpl': 'DBD-SQLite-1.55_03.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/I/IS/ISHIGAKI'],
+        'checksums': [
+            'ac22b9356e51ee32559d315aa383916dfd39b142cf45824284a495a287f1f893',  # DBD-SQLite-1.55_03.tar.gz
+        ],
+    }),
+    ('Date::Language', '2.30', {
+        'source_tmpl': 'TimeDate-2.30.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GB/GBARR'],
+        'checksums': [
+            '75bd254871cb5853a6aa0403ac0be270cdd75c9d1b6639f18ecba63c15298e86',  # TimeDate-2.30.tar.gz
+        ],
+    }),
+    ('Template', '2.27', {
+        'source_tmpl': 'Template-Toolkit-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AB/ABW'],
+        'checksums': [
+            '1311a403264d0134c585af0309ff2a9d5074b8ece23ece5660d31ec96bf2c6dc',  # Template-Toolkit-2.27.tar.gz
```

Comment:
@smoors A more recent implementation of `--inject-checksums` does not include comments when there's only a single source, please re-run?